lib/node_modules/@stdlib/stats/base/dists/negative-binomial/ctor/benchmark/benchmark.js

```diff
@@ -90,12 +99,16 @@ bench( pkg+'::set:r', function benchmark( b ) {
 	r = 40;
 	p = 0.6;
 	dist = new NegativeBinomial( r, p );
+	len = 100;
+	y = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		y[ i ] = discreteUniform( 1.0, 100.0 );
```

Comment:
```suggestion
		y[ i ] = discreteUniform( 1, 50 );
```

Make sure the range is still the same for consistency.